package_manifest.yaml

```diff
@@ -269,25 +265,25 @@ katello_packages:
     rubygem-robotex: {}
     rubygem-runcible: {}
     rubygem-terminal-table: {}
-katello_client_packages:
   vars:
-    package_base_dir: 'packages/katello/'
+    package_base_dir: packages/katello/
     releasers:
-    - koji-katello-client
-  hosts:
-    katello-host-tools: {}
-    katello-repos: {}
+    - koji-katello
+packages:
```

Comment:
I kinda liked this being at the top as it gave a general overview of the groups I expect to find below.

Reply:
Perhaps this can be a convention that is always at the top?

Reply:
I like that too and will see how I can force the formatter to behave.